src/Service/Startup.cs

```diff
@@ -277,7 +277,6 @@ private void AddGraphQLService(IServiceCollection services, GraphQLRuntimeOption
                 })
                 .AddHttpRequestInterceptor<IntrospectionInterceptor>()
                 .AddAuthorization()
-                .AllowIntrospection(false)
```

Comment:
`AllowIntrospection` is no longer needed. With HC 14 we have this as a default for production environment. There are a couple of other changes around this that we should discuss.